src/config/IAppConfig.ts

```diff
@@ -190,4 +190,7 @@ export interface IServerConfig {
     clickhouse_mode: boolean;
     download_custom_buttons_json: string;
     feature_study_export: boolean;
+    skin?: {
```

Comment:
we have a convention above that dot notation gets converted to _.  lets keep that

Reply:
if you test it this, the backend should automatically convert the dot notation.  so i would think this wouldn't work.   let me know if you need hlep testing this